src/mono/mono/mini/mini-wasm-debugger.c

```diff
@@ -360,6 +363,7 @@ mono_wasm_debugger_init (void)
 	mono_profiler_set_jit_done_callback (prof, jit_done);
 	//FIXME support multiple appdomains
 	mono_profiler_set_domain_loaded_callback (prof, appdomain_load);
+	mono_profiler_set_assembly_loaded_callback(prof, assembly_load);
```

Comment:
nit:
* mono uses tabs, not spaces, with tab width = 8
* space before '('
```suggestion
	mono_profiler_set_assembly_loaded_callback (prof, assembly_load);
```